src/App.css

```diff
@@ -13,21 +13,23 @@ body {
 
```

Comment:
### Your Design
![image](https://user-images.githubusercontent.com/93039359/234454307-9b9c20b4-b0ef-4fd2-8196-49b01608f99a.png)

### Expected Design
![image](https://user-images.githubusercontent.com/93039359/234454359-67ebeac3-4fa1-413a-ae4d-34a9b5116f84.png)

- [ ] It would be nice if you follow the design to make your project look attractive and professional as suggested by the previous reviewer :+1: because right now I have seen a lot of things which still need more attention :fearful:
* First the calculator should have more width than height as you can see in the expected design
* Second I saw a black border line around buttons which is not really visible in the expected design.
* Last, The screen of the calculator should have same width, height and the zero **(0)** should be on the right side not on the left side of the screen :+1:

Kindly make all these changes to make your project look professional :blush: